src/amo/components/PermissionsCard/permissions.js

```diff
@@ -59,28 +61,67 @@ export class PermissionUtils {
       topSites: i18n.gettext('Access browsing history'),
       webNavigation: i18n.gettext('Access browser activity during navigation'),
     };
+
+    this.dataCollectionPermissionStrings = {
```

Comment:
Curious why we are defining these strings on addons-frontend?

Just easier I assume but I guess if we needed these elsewhere we would either migrate to addons-server or copy/pasta?

Reply:
I'm following what we did for regular permissions. Historically we went back and forth but ultimately settled on having them on frontend only as we didn't need to display them elsewhere (reviewer tools can display the permission name, that's good enough)